tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -7346,4 +7354,24 @@ public static String utf8truncate(String input, int length) {
 	return result.toString();
     }
 
+	/**
+	 * Method to add Resources folder into Lessons tool
+	 */
+	public String folderPickerSubmit(){
+		//Check if user has submitted page without selecting folder?
+		String defaultPath = "/group/" + getCurrentSiteId();
```

Comment:
Rather than hard coding this you should use `org.sakaiproject.content.api.ContentHostingService#getSiteCollection`